### DISCUSSION TITLE
Support Allure reports

### DISCUSSION THREAD

**Discussion by daniil00d** [storybookjs/storybook#26508] [Help]
### Summary

Hello! We are using Storybook and writing interactive tests for our stories, and then running the tests from our CI. We noticed that under the hood, Storybook test runners use Playwright. Is there a way to generate Allure reports and parameterize the Playwright functions (such as Feature, Issue, TestId, etc.)?

**Comment by moritz-baecker-integra** [comment]
That would be really helpful to know. Did you find out about this?

**Comment by VictorDanilov** [comment]
Any news about this question?
